### PR TITLE
Some docs optimization

### PR DESCRIPTION
**Pull request by zhaojiandong**
Some docs optimization

### CODE REVIEW

File: docs/content/operations/metrics.md [hunk 1]
```diff
@@ -244,9 +244,7 @@ The following metric is only available if the EventReceiverFirehoseMonitor modul
 
 |Metric|Description|Dimensions|Normal Value|
 |------|-----------|----------|------------|
-|`ingest/events/buffered`|Number of events queued in the EventReceiverFirehose's buffer|serviceName, dataSource, taskId, taskType, bufferCapacity
-.|Equal
-to current # of events in the buffer queue.|
+|`ingest/events/buffered`|Number of events queued in the EventReceiverFirehose's buffer|serviceName, dataSource, taskId, taskType, bufferCapacity.|Equalto current # of events in the buffer queue.|
```
Comment (gianm) [inline docs/content/operations/metrics.md]: There should be a space between "Equal" and "to".

File: docs/content/operations/api-reference.md [hunk 2]
```diff
@@ -526,8 +526,8 @@ druid.query.segmentMetadata.defaultHistory
 Returns the dimensions of the datasource.
 
 <div class="note caution">
-This API is deprecated and will be removed in future releases. Please use [SegmentMetadataQuery](../querying/segmentmetadataquery.html) instead
-which provides more comprehensive information and supports all dataSource types including streaming dataSources. It's also encouraged to use [INFORMATION_SCHEMA tables](../querying/sql.html#retrieving-metadata)
+This API is deprecated and will be removed in future releases. Please use <a href="/querying/segmentmetadataquery.html">SegmentMetadataQuery</a> instead
```
Comment (gianm) [inline docs/content/operations/api-reference.md]: The `[text](url)` syntax should work fine and is standard for Markdown. Was something not working with it? (I notice one of the links had a link break between `[SegmentMetadataQuery]` and `(../querying/segmentmetadataquery.html)`, that one probably needed to be connected.

Comment (zhaojiandong) [reply docs/content/operations/api-reference.md]: Maybe github hasn't supported markdown="1" :
```
<div markdown="1">
    #Heading 1
</div>
```
It's not work @gianm

File: docs/content/querying/sql.md [hunk 3]
```diff
@@ -149,7 +149,7 @@ String functions accept strings, and return a type appropriate to the function.
 
 |Function|Notes|
 |--------|-----|
-|`x \|\| y`|Concat strings x and y.|
+|`x &#124;&#124; y`|Concat strings x and y.|
```
Comment (jon-wei) [inline docs/content/querying/sql.md]: Hm, I don't think these escapes are working: https://github.com/apache/incubator-druid/blob/1d100a54f70c00729f0b33d32ae6a2a042b48cb7/docs/content/querying/sql.md

File: docs/content/operations/api-reference.md (hunk 2 above)
Comment (gianm) [reply docs/content/operations/api-reference.md]: Oh, I guess inside HTML, markdown isn't respected? I didn't realize that.

File: docs/content/querying/sql.md (hunk 3 above)
Comment (gianm) [reply docs/content/querying/sql.md]: After playing around a bit I found something that works:

```
|<code>x &#124;&#124; y</code>|Concat strings x and y.|
```

File: docs/content/operations/api-reference.md (hunk 2 above)
Comment (zhaojiandong) [reply docs/content/operations/api-reference.md]: I guess so~

File: docs/content/querying/sql.md (hunk 3 above)
Comment (zhaojiandong) [reply docs/content/querying/sql.md]: It works! :+1:

Comment (zhaojiandong) [reply docs/content/querying/sql.md]: I found it works in https://github.com/zhaojiandong/druid/blob/master/docs/content/querying/sql.md
not work in https://github.com/apache/incubator-druid/blob/57d95fe194df9af00687203950fa86a2fe9e025d/docs/content/querying/sql.md

**Pull request merged**